test/state/state.hpp

```diff
@@ -24,6 +25,21 @@ class State
         assert(r.second);
         return r.first->second;
     }
+
+    /// Get an account from the address
+    Account& get(const address& addr) { return m_accounts.at(addr); }
+};
+
+struct ClientInfo
+{
+    std::string comment;
+    std::string rpc_server_version;
+    std::string tool_version;
+    std::string lllc_version;
+    hash256 generated_test_hash;
+    std::string source;
+    hash256 source_hash;
+    std::unordered_map<uint64_t, std::string> labels;
```

Comment:
I think we are not interested in other metadata than input labels for now so try to place this directly in `class StateTranstionTest`.  This can also be done separately from this PR for existing state test loader.